### DISCUSSION TITLE
Use Windows line endings for resx files

### DISCUSSION THREAD

**Discussion by rgroenewoudt** [WeblateOrg/weblate#10411] [Q&A]
We are using Weblate on a Linux docker machine to manage our .NET translations, which are in .resx format.
The problem is that Weblate commits the file to Git with unix newlines and not windows/dos newlines (CRLF). We are talking about the resx file itself, not only the translation inside it.

Is there a way to configure Weblate to use CRLF?

**Comment by nijel** [comment]
There is currently no way to configure this. If these files typically use CRLF, please open issue at https://github.com/translate/translate/ to make it behave like that.

**Reply by rgroenewoudt** [reply]
https://github.com/translate/translate/issues/5119
It has been fixed :)
